docs/core/porting/premigration-needed-changes.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Needed changes before porting code
+description: Make the necessary changes to build targeting .NET Core.
```

Comment:
Again, these should be pretty descriptive. Also, I wonder if a more action-focused title would be good? Maybe it sounds too blog-like?
```suggestion
title: Do these things before porting code
description: Upgrade your dev environment to the latest version, and upgrade your project files. You must do these things before porting your code from .NET Framework.
```